students/byeongyeon/westagram/users/views.py

```diff
@@ -40,3 +40,18 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message": "KEY_CRROR"}, status=400)
 
+class LogInView(View):
+    def post(self, request):
+        data     = json.loads(request.body)
+        email    = data["email"]
+        password = data["password"]
+
+        try:
+            if not User.objects.filter(email=email, password=password).exists():
+                return JsonResponse({"message": "INVALID_USER"}, status=401)
+            else:
+                return JsonResponse({"message": "SUCCESS"}, status=200)
```

Comment:
else가 없어도 충분히 잘 동작합니다. 왜 그런지 생각해 보시면 좋을 것 같아요
```suggestion
            
            return JsonResponse({"message": "SUCCESS"}, status=200)
```